app-text/xpdf/xpdf-4.04-r1.ebuild

```diff
@@ -47,7 +47,6 @@ DEPEND="
 	sys-libs/zlib
 "
 RDEPEND="${DEPEND}
-	dev-qt/qtsvg:5
```

Comment:
?

this isn't mentioned in the commit message and also needs a new revision

Reply:
If removing it because of lack of DT_NEEDED or reference in CMakeLists.txt, note that it can often be needed even if not linked with it for qtsvg (it will be loaded at runtime if needed). I would try running it without qtsvg installed to see if there's anything odd (like missing icons, I notably see that it has many .svg in its xpdf-qt/ directory).

Reply:
There is no reference to `QtSVG` anywhere in the code. The package ships `svg` icons, but that does not create a dependency on `QtSVG`. Only for the Packages that would want to make use of those icons.

If this is the only package pulling in `qtsvg:5` the package manager should be able to reinstall this to be able to uninstall `qtsvg:5` anyway.

Reply:
It also compiles and runs without `qtsvg` installed.

Reply:
It's a runtime-only dependency so of course it would.

Did you try running it without qtsvg? It does not need to have a reference to it to be needed sometimes.

Reply:
Most case it's needed but missing, it results in missing UI elements.

Reply:
Also in case missed the edit, I had added:
>Edit: and not talking of the one that's converted with rsvg-convert

Not talking about the deskop file icons, but the in-application icons.

Reply:
Menu Icons are missing without `qtsvg:5`. Hence dropped that part. Will document later when I find out what part is needed from texlive.